variables.tf

```diff
@@ -0,0 +1,206 @@
+  type        = bool
+  default     = true
+}
+variable "subnet_ids" {
+  description = "(Optional) the subnet IDs which the subnet group is going to be attach, only required if you don't specify a value for var.db_subnet_group"
+  type        = list(string)
+  default     = []
+}
+
+variable "enabled_cloudwatch_logs_exports" {
+  description = "List of log types to enable for exporting to CloudWatch logs. If omitted, no logs will be exported. Valid values (depending on engine): alert, audit, error, general, listener, slowquery, trace, postgresql (PostgreSQL), upgrade (PostgreSQL)."
+  type        = list(string)
+  default     = []
+}
+
+variable "engine" {
+  description = "The database engine to use"
+  type        = string
+  default     = "MYSQL"
+}
+
+variable "engine_version" {
+  description = "The engine version to use"
+  type        = string
+  default     = "8.0.13"
+}
+
+variable "iam_database_authentication_enabled" {
+  description = "Specifies whether or not the mappings of AWS Identity and Access Management (IAM) accounts to database accounts are enabled"
+  type        = bool
+  default     = false
+}
+
+variable "publicly_accessible" {
+  description = "Boolean to control if instance is publicly accessible"
+  type        = bool
+  default     = false
+}
+
+variable "instance_class" {
+  description = "The instance class for your instance(s), by default we are using db.t2.small because we want to enable encryption, if you disable encryption you can choose  db.t2.micro"
+  type        = string
+  default     = "db.t2.small"
+}
+
+variable "kms_key_id" {
+  description = "The ARN for the KMS encryption key. If creating an encrypted replica, set this to the destination KMS ARN. If storage_encrypted is set to true and kms_key_id is not specified the default KMS key created in your account will be used"
+  type        = string
+  default     = ""
+}
+
+variable "maintenance_window" {
+  description = "The window to perform maintenance in. Syntax: 'ddd:hh24:mi-ddd:hh24:mi'. Eg: 'Mon:00:00-Mon:03:00'"
+  type        = string
+  default     = "Sun:00:00-Sun:04:00"
+}
+
+variable "monitoring_interval" {
+  description = "The interval, in seconds, between points when Enhanced Monitoring metrics are collected for the DB instance. To disable collecting Enhanced Monitoring metrics, specify 0. The default is 0. Valid Values: 0, 1, 5, 10, 15, 30, 60."
+  type        = number
+  default     = "0"
+}
+
+variable "monitoring_role_arn" {
+  description = "The ARN for the IAM role that permits RDS to send enhanced monitoring metrics to CloudWatch Logs. Must be specified if monitoring_interval is non-zero."
+  type        = string
+  default     = ""
+}
+
+variable "availability_zone" {
+  description = "Only set if multi_az is in the default seeting (false) for multi_az = true leave this blank"
+  type        = string
+  default     = ""
+}
```

Comment:
Change the spelling "the default seeting" to "the default setting.."
Is this variable required? If it is optional I remember you can set default = null